subprojects/packagefiles/eyalroz-printf/printf-6.2.0/test/meson.build

```diff
@@ -0,0 +1,95 @@
+tests_c_args = cc.get_supported_arguments(
```

Comment:
Does the get_supported_arguments() support list of strings as an argument? In the documentation only variadic string

> The method accepts between 0 and infinity variadic arguments (arg...) of type [str](https://mesonbuild.com/Reference-manual_elementary_str.html)